utility/cmake_fc_config.h.in

```diff
@@ -57,8 +57,6 @@
 /* memory.h available */
 #cmakedefine HAVE_MEMORY_H
 
-/* pwd.h available */
-#cmakedefine HAVE_PWD_H
```

Comment:
There must be `HAVE_GETPWUID` somewhere, you removed it from `FindDependencies` so it can be dropped too.